src/Catalyst.Core.Modules.Ledger/Ledger.cs

```diff
@@ -256,5 +296,116 @@ public void Dispose()
             Dispose(true);
             GC.SuppressFinalize(this);
         }
+
+        class ReceiptDeltaTracer : ITxTracer
+        {
+            readonly Delta _delta;
+            readonly long _deltaNumber;
+            readonly IHashProvider _hashProvider;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this unread private field '_hashProvider' or refactor the code to use its value.](https://app.codacy.com/manual/catalyst-network/Catalyst.Framework/pullRequest?prid=4585091)